README.md

```diff
@@ -395,25 +461,30 @@ npm test
 # npm run test-windows
 ```
 
-##### Attribution
-We have an automatic build setup on Travis CI that will run every day to catch changes made by Google as quick as possible. These tests make use of BrowserStack's services to remotely run Selenium tests, huge thanks to [BrowserStack][] for the support of Open Source projects like this one!
+#### Attribution
```

Comment:
While were here, this should probably be removed.  We no longer test nightly as we don't have any of the infrastructure set up to do so.

Reply:
Working on it